pkg/core/object/metadata.go

```diff
@@ -568,7 +568,9 @@ func matchValues(dbVal []byte, matcher object.SearchMatchType, fltVal []byte) bo
 	}
 }
 
-func restoreIntAttribute(b []byte) (string, error) {
+// RestoreIntAttribute restores from raw signed uint256 format its string
+// representation.
+func RestoreIntAttribute(b []byte) (string, error) {
```

Comment:
exporting should never be done for tests imo

Reply:
this code is used 3 times, do you mean making 3 different copy of it?